syntax = "proto3";

import "hq/dto/Security.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.hq.request";
option java_outer_classname = "OrderBookRequestProto";
message OrderBookRequest {
	Security security = 1;  // 股票定义
	int32 mktTmType = 2; // -1 1 -2 盘前 盘中 盘后
	int32 depthBookType = 3;	// 2 3 totalview arcabook （非必填）
}
